src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -161,6 +161,36 @@ export class SkyCoreAdapterService {
     return elements;
   }
 
+  /**
+   * Checks if an event target has a higher z-index than a given element.
+   * @param target The event target element.
+   * @param elementRef The element to test against. A z-index must be explicitly set for this element.
+   */
+  public isTargetAboveElement(target: EventTarget, element: HTMLElement): boolean {
```

Comment:
Would this be more reusable if the `target` argument was an `HTMLElement`?

Reply:
I was hoping to communicate with the method parameters that the first "element" should always be the event's "target", not just some element.

Reply:
Got it. Maybe deliberate is better in this situation. I see you also spell that out in the description. I did notice your jsdoc variable is wrong though `elementRef` --> `element`.